src/components/WebexAvatar/README.md

```diff
@@ -1,26 +1,31 @@
 # Webex Avatar Component
+
 Webex avatar component displays the avatar and status of a _Webex Teams_ user.
 
 <p align="center">
   <img src="./WebexAvatar.png" alt="Default Webex Avatar" />
 </p>
 
 ## Preview
+
 To see all the different possible states of the Webex Avatar component, you can run our Storybook:
+
 ```shell
   npm start
 ```
 
 ## Embed
+
 1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
 
-    ```js
-      const jsonAdapter = new PeopleJSONAdapter();
-    ```
+   ```js
+   const adapter = new PeopleJSONAdapter(people);
```

Comment:
don't change the variable name, I think is better to have it explicitly say "json" as its type